src/Login.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import "./pages/Subpage.css"
 
 function Login() {
     return (
         <div className="App">
-            <header className="App-header">
-                <a className="btn-spotify" href="/auth/login" >
-                    Login with Spotify 
-                </a>
-            </header>
+            <header className="App-header" class="font_one"><b>
+                Login with Spotify 
+            </b></header>
+            <a className="btn-spotify" href="/auth/login" >
+                    <img src="spoootify.png" alt="spotify" height="70"/>
+            </a>
```

Comment:
I like this login implementation a lot better than the previous one. Good job coming up with this idea!

Reply:
Thank you! I appreciate it.